client/layout/nps-survey-notice/index.jsx

```diff
@@ -95,6 +130,8 @@ const mapStateToProps = state => {
 		hasAnsweredWithNoScore: hasAnsweredNpsSurveyWithNoScore( state ),
 		isSectionAndSessionEligible: isSectionAndSessionEligibleForNpsSurvey( state ),
 		wasShownThisSession: wasNpsSurveyShownThisSession( state ),
+		npsSurveyScore: getNpsSurveyScore( state ),
+		isBusinessUser: getSites( state ).filter( isOwnBusinessSite ).length > 0,
```

Comment:
`getSites( state ).some( isOwnBusinessSite )` is slightly more efficient: returns results as soon as it finds the first matching site. And returns the desired boolean value straight away.

Reply:
@jsnajdr Thanks for your review. Now the new API endpoint for counting concierge sessions has been dropped. Instead, the existing NPS API endpoint contains additional prop `has_available_concierge_sessions`.

Reply:
So there's also a wpcom patch that adds the new prop? Thanks, that addresses all my concerns about this PR 🙂 :+1: